doc/ADR/homebrew_tap.md

```diff
@@ -1,5 +1,12 @@
 # Homebrew tap instructions
 
+CBMC archives versions into a tap which you can use to quickly download and
+build various historical versions.
+
+The tap is available at https://github.com/diffblue/homebrew-cbmc.
+
+Any version installed from the tap is going to be built from source.
```

Comment:
Suggestion - Replace "is going to be built from source." -> "will need to be built from source locally." Technically all versions are built from source at some point in the process, even if that is done as part of our process to provide pre-built binaries.